apps/bbsync/tests/test_query.py

```diff
@@ -248,32 +241,23 @@ def test_generate_acknowledgments(self):
         """
         flaw = FlawFactory(source=FlawSource.CUSTOMER)
         FlawCommentFactory(flaw=flaw)
-        FlawMetaFactory(flaw=flaw, type=FlawMeta.FlawMetaType.REFERENCE)  # not an ack
-        FlawMetaFactory(
+        FlawAcknowledgmentFactory(
             flaw=flaw,
-            type=FlawMeta.FlawMetaType.ACKNOWLEDGMENT,
-            meta_attr={
-                "affiliation": "dear",
-                "from_upstream": False,
-                "name": "sir",
-            },
+            affiliation="dear",
+            from_upstream=False,
+            name="sir",
         )
-        FlawMetaFactory(
+        FlawAcknowledgmentFactory(
             flaw=flaw,
-            type=FlawMeta.FlawMetaType.ACKNOWLEDGMENT,
-            meta_attr={
-                "affiliation": "von Bahnhof",
-                "from_upstream": True,
-                "name": "Carl",
-            },
+            affiliation="von Bahnhof",
+            from_upstream=True,
+            name="Carl",
```

Comment:
`name` and `affiliation` are probably wrong. `name` can be e.g. "Carl von Bahnhof" and `affiliation` can be e.g. "Google Security Team". (I think "von Bahnhof" is a part of the name, not an organization. This problem occurs more times.)

Reply:
Added names that should be more descriptive for demo purposes.